test/session_spec.lua

```diff
@@ -102,7 +102,7 @@ function test_session(description, session_factory, session_destroy)
       local requested = 0
       local _, channel_id, _ = get_api_info(session)
 
-      local function on_request(method, args)
+      local function on_request(method)
```

Comment:
i think the `args` param should be fake-used. This is a callback that requires a particular signature IIRC.

Reply:
That would be odd though.. do you mean it gets inspected somewhere really?
Because in general this works:
```lua
> f = function(foo) print(foo) end
> f(foo, "bar")
nil
> f("foo", "bar")
foo
```

Reply:
> Because in general this works:

Oh, I should have tested but guess I forgot about that Lua feature...